src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -4889,10 +4889,13 @@ static void Main(string[] args)
                 VerifyDiagnostics(
                 // (9,26): error CS0157: Control cannot leave the body of a finally clause
                 //         finally { i = 3; goto lab1; }// invalid
-                Diagnostic(ErrorCode.ERR_BadFinallyLeave, "goto"),
+                Diagnostic(ErrorCode.ERR_BadFinallyLeave, "goto").WithLocation(9, 26),
+                // (10,5): warning CS0162: Unreachable code detected
+                //     lab1:
+                Diagnostic(ErrorCode.WRN_UnreachableCode, "lab1").WithLocation(10, 5),
```

Comment:
>Diagnostic(ErrorCode.WRN_UnreachableCode, "lab1").WithLocation(10, 5), [](start = 16, length = 70)

This warning looks strange

Reply:
Will revert changes that cause this.

---
In reply to: [151040756](https://github.com/dotnet/roslyn/pull/23172#discussion_r151040756) [](ancestors = 151040756)